src/render/attrs/index.js

```diff
@@ -10,6 +10,42 @@ const attrsNotToRender = {
   dangerouslySetInnerHTML: true
 };
 
+const attrsWithExplicitBoolValue = {
+  "aria-expanded": true,
+  "aria-haspopup": true
+};
```

Comment:
If there's something in React we can use for this logic, that sounds great. I don't quite understand the function you linked to. Could you show me how I might use it to replace my `attrsWithExplicitBoolValue` please?

I don't want to maintain a a list, but I don't currently understand what the logic is that would make the list unnecessary.

@mijamo